svgnative/src/SVGNativeCWrapper.cpp

```diff
@@ -239,20 +239,25 @@ void svg_native_render_size(svg_native_t* sn, float width, float height)
 }
 
 #ifdef USE_TEXT
-void svg_native_render_output(svg_native_t* sn, char* output, int& length)
+size_t svg_native_render_output(svg_native_t* sn, char* buff, size_t buff_size)
```

Comment:
I wrote this some time ago. Looking at it again we should probably use something like [cairo-surface-get-mime-data](https://www.cairographics.org/manual/cairo-cairo-surface-t.html#cairo-surface-get-mime-data).

```c
const unsigned char* data;
unsigned long length;

cairo_surface_get_mime_data (surface, mime_type, &data, &length);
```

Reply:
Using size_t instead of unsigned long would be fine with me if that gets accepted by C compilers.

Reply:
I understand you suggest making test program emits rasterized image, instead of text info, I agree, and will do it.

Reply:
At 5288828 , still Text port is selectable, how do you think? Should I remove it completely?